### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

File: docs/zh_CN/FrameworkControllerMode.md [hunk 1]
```diff
@@ -102,4 +102,8 @@ frameworkcontroller 模式中的 Trial 配置使用以下主键：
 
 ## 如何运行示例
 
-准备好配置文件后，通过运行 nnictl 来启动 Experiment。 在 Frameworkcontroller 上开始 Experiment 的方法与 Kubeflow 类似，可参考[指南](./KubeflowMode.md)了解更多信息。
\ No newline at end of file
+准备好配置文件后，通过运行 nnictl 来启动 Experiment。 在 Frameworkcontroller 上开始 Experiment 的方法与 Kubeflow 类似，可参考[指南](./KubeflowMode.md)了解更多信息。
+
+## 版本校验
+
+从 0.6 开始，NNI 支持查看版本，详情参考[这里](PAIMode.md)。
```
Comment (scarlett2018) [inline docs/zh_CN/FrameworkControllerMode.md]: how about use https://github.com/Microsoft/nni/blob/56ac616f4a777a15b64e35670293d21cc693ddf1/docs/zh_CN/PAIMode.md#%E7%89%88%E6%9C%AC%E6%A0%A1%E9%AA%8C instead.

File: docs/zh_CN/KubeflowMode.md [hunk 2]
```diff
@@ -202,4 +202,8 @@ Kubeflow 模式的配置有下列主键：
 
 当一个 Trial 作业完成后，可以在 NNI 网页的概述页面（如：http://localhost:8080/oview）中查看 Trial 的信息。
 
+## 版本校验
+
+从 0.6 开始，NNI 支持版本校验，详情参考[这里](PAIMode.md)。
```
Comment (scarlett2018) [inline docs/zh_CN/KubeflowMode.md]: same comments as above

File: docs/zh_CN/PAIMode.md [hunk 3]
```diff
@@ -82,4 +82,16 @@ paiConfig:
 
 如果希望将 Trial 的模型数据等其它输出保存到HDFS中，可在 Trial 代码中使用 `NNI_OUTPUT_DIR` 来自己保存输出文件，NNI SDK会从 Trial 的容器中将 `NNI_OUTPUT_DIR` 中的文件复制到 HDFS 中。
 
-如果在使用 pai 模式时遇到任何问题，请到 [NNI Github](https://github.com/Microsoft/nni) 中创建问题。
\ No newline at end of file
+如果在使用 pai 模式时遇到任何问题，请到 [NNI Github](https://github.com/Microsoft/nni) 中创建问题。
+
+## 版本校验
```
Comment (scarlett2018) [inline docs/zh_CN/PAIMode.md]: I'm actually a little confused, why version check is in the PAIMode.md, this sounds more like a generic feature in NNI.

File: docs/zh_CN/FrameworkControllerMode.md (hunk 1 above)
Comment (squirrelsc) [reply docs/zh_CN/FrameworkControllerMode.md]: you can change en-us first, and it will be in zh-cn.

File: docs/zh_CN/PAIMode.md (hunk 3 above)
Comment (squirrelsc) [reply docs/zh_CN/PAIMode.md]: I raised this in chat group, please follow up it..

**Pull request merged**